OpenRA.Game/Widgets/ChatDisplayWidget.cs

```diff
@@ -47,7 +47,7 @@ public override void Draw()
 				}
 
 				var text = WidgetUtils.WrapText(line.Text, chatLogArea.Width - inset - 6, font);
-				chatpos.Y -= Math.Max(15, font.Measure(text).Y) + 5;
+				chatpos = chatpos.WithY(chatpos.Y - Math.Max(15, font.Measure(text).Y) + 5);
```

Comment:
May be just missing a bracket now. Try `chatpos = chatpos.WithY(chatpos.Y - (Math.Max(15, font.Measure(text).Y) + 5));`